pkg/repository/prisma/job_run.go

```diff
@@ -35,8 +38,26 @@ func NewJobRunAPIRepository(client *db.PrismaClient, pool *pgxpool.Pool, v valid
 	}
 }
 
+func (j *jobRunAPIRepository) RegisterWorkflowRunRunningCallback(callback repository.Callback[pgtype.UUID]) {
```

Comment:
Is this Workflow Run Callback stuff related to the metrics ?

Reply:
Yes, we register callbacks so that when we wrap the engine with our own custom code, we can get events into the metrics database.